### PR TITLE
Refactor/locales

### PR DESCRIPTION
**Pull request by queuing4oranges**
- replacing dynamic values in locales - in asab-webui

-- when locales is loaded normally:

https://github.com/TeskaLabs/asab-webui/assets/31582696/7b0d5275-1eb5-4842-b4be-1a1bacbbb649

-- when the locales file is corrupted:

https://github.com/TeskaLabs/asab-webui/assets/31582696/a1aff49c-d8d9-4f37-995f-6a1628e97140

### CODE REVIEW

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@queuing4oranges Ok, this looks good - dont forget to update also locales in `demo` app and add the changelog. Btw. the videos can be displayed because of corrupted content :)

**Comment by queuing4oranges** [comment]
replacing locales in asab-webui `demo` app

when locales load normally:

https://github.com/TeskaLabs/asab-webui/assets/31582696/94de72ef-ebaf-4cd8-93cb-6316948691f1

when locales file is corrupted:


https://github.com/TeskaLabs/asab-webui/assets/31582696/e2212610-80f5-4d85-aca9-da2f33693378


hope the video works!

File: src/components/Pagination.js [hunk 1]
```diff
@@ -162,7 +162,8 @@ export default function ({ currentPage, setPage, lastPage, style }) {
 			</Dropdown>
 
 			<div className="mr-2">
-				{pages && (pages.length > 1) ? t('Pages', {pages: (lastPage == 0) ? 1 : lastPage}) : t('Page', {pages: (lastPage == 0) ? 1 : lastPage})}
+				{pages && (pages.length > 1) ? (`${t("of")} ${pages.length} ${t("Pages")}`) :
+				(`${t("of")} ${pages.length} ${t("Page")}`)}
```
Comment (Pe5h4) [inline src/components/Pagination.js:166]: @queuing4oranges Here I believe we want to have sometjing like:

```
{pages && (pages.length > 1) ? (`${t("of")} ${(lastPage == 0) ? 1 : lastPage} ${t("Pages")}`) : (`${t("of")} ${(lastPage == 0) ? 1 : lastPage} ${t("Page")}`)}
```

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@queuing4oranges Small change request

**Pull request merged**